api/src/test/java/org/openmrs/module/patientflags/FlagTest.java

```diff
@@ -173,49 +137,23 @@ public void eval_groovyShouldReturnFalseForTestPatient() throws Exception {
 	
 	// TODO add tests that actually create non-empty result sets
 	// TODO add tests to verify what evalCohort does if invalid evaluator
-
-	@Test
-	public void evalCohort_logicShouldReturnEmptyCohort() throws Exception {
-		Flag flag = Context.getService(FlagService.class).getFlag(2);
-		flag.setEvaluator(PatientFlagsConstants.FLAG_EVALUATOR_MAP.get("logic"));
-		Cohort cohort = flag.evalCohort(new Cohort());
-		Assert.assertTrue(cohort.isEmpty());
-	}
 	
 	@Test
 	public void evalCohort_sqlShouldReturnEmptyCohort() throws Exception {
-		Flag flag = Context.getService(FlagService.class).getFlag(4);
+		Flag flag = Context.getService(FlagService.class).getFlag(1);
 		flag.setEvaluator(PatientFlagsConstants.FLAG_EVALUATOR_MAP.get("sql"));
 		Cohort cohort = flag.evalCohort(new Cohort());
 		Assert.assertTrue(cohort.isEmpty());
 	}
 	
-	// TODO: get tests that require logic module to be running to work
-	//@Test
-	//public void evalCohort_logicNullShouldReturnEmptyCohort() throws Exception {
-		//Flag flag = Context.getService(FlagService.class).getFlag(2);
-		//flag.setEvaluator(PatientFlagsConstants.FLAG_EVALUATOR_MAP.get("logic"));
-		//Cohort cohort = flag.evalCohort(null);
-		//Assert.assertTrue(cohort.isEmpty());
-	//}
-	
 	@Test
 	public void evalCohort_sqlNullShouldReturnCohort() throws Exception {
-		Flag flag = Context.getService(FlagService.class).getFlag(4);
+		Flag flag = Context.getService(FlagService.class).getFlag(1);
 		flag.setEvaluator(PatientFlagsConstants.FLAG_EVALUATOR_MAP.get("sql"));
 		Cohort cohort = flag.evalCohort(null);
 		Assert.assertTrue(cohort.isEmpty());
 	}
 	
-	
-	
-	/**
-	 * Mimics org.openmrs.web.Listener.getRuntimeProperties()
-	 * 
-	 * @param webappName name to use when looking up the runtime properties env var or filename
-	 * @return Properties runtime
-	 * @throws Exception 
-	 */
     @SuppressWarnings("deprecation")

```

Comment:
@pgesek When I run `mvn clean install` I get stuck at running the FlagTest, I suspect the issue may be with the datasets for the different build versions. However the datasets are all the same so why not just import a single XML dataset and delete this function

Reply:
@pgesek did you see the above? 😊

Reply:
@pgesek just a ping to bring this back to top of mind

Reply:
@pgesek Happy Monday, ping on this - thanks

Reply:
@ssmusoke I wasn't able to reproduce this hanging issue, perhaps it has something to do with the database connection? I've removed the redundant as you suggested however.